src/main.js

```diff
@@ -7,13 +7,33 @@ const getLogger = () => {
 		return new FunctionLogger();
 	} else {
 		// app environment - use Winston
+
+		// Determine whether the app is using Heroku log drains
+		const useHerokuLogDrains = (
+			process.env.MIGRATE_TO_HEROKU_LOG_DRAINS &&
+			process.env.MIGRATE_TO_HEROKU_LOG_DRAINS.length > 0
+		);
```

Comment:
**nitpick**: an empty string is falsy, so the `length > 0` check is redundant

Reply:
Oh yeah 😄 I copied this pattern from a few lines down without thinking, I'll correct both